src/app/screens/PeerlyScreens/AppreciationScreen/index.tsx

```diff
@@ -1,9 +1,217 @@
-import {Text, View} from 'react-native';
+import React, {useState} from 'react';
+import {
+  Text,
+  View,
+  StyleSheet,
+  TextInput,
+  Button,
+  SafeAreaView,
+  Pressable,
+} from 'react-native';
+import {
+  useGetCoworkerList,
+  useGetCoreValuesList,
+  usePostAppreciation,
+} from './appreciation.hooks';
+import Typography from '../../../components/typography';
+import {useForm, Controller, SubmitHandler} from 'react-hook-form';
+import Select from '../Components/Select';
+import CenteredModal from '../Components/Modal';
+import {ScrollView} from 'react-native-gesture-handler';
+import * as yup from 'yup';
+import {yupResolver} from '@hookform/resolvers/yup';
+import CoreValueInfoModal from '../Components/CoreValueInfoModal';
+import {FormInput} from './types';
 
-export default function AppreciationScreen() {
+import SuccessIcon from '../../../../assets/peerly/svg/Vector.svg';
+import InfoIcon from '../../../../assets/peerly/svg/info.svg';
+
+const paginationData = {
+  page: 1,
+  per_page: 500,
```

Comment:
Please use camel case.

Reply:
This is pagination key values need to BE in snake format only that's why we used it. it's not for FE purpose.